docs/guide/essentials/content-scripts.md

```diff
@@ -205,14 +205,13 @@ export default defineContentScript({
   matches: ['<all_urls>'],
 
   main(ctx) {
+    let app: Record<string, any>;
     const ui = createIntegratedUi(ctx, {
       position: 'inline',
       anchor: 'body',
       onMount: (container) => {
         // Create the Svelte app inside the UI container
-        mount(App, {
-          target: container,
-        });
+        app = mount(App, { target: container });
```

Comment:
```suggestion
        const app = mount(App, { target: container });
        return app;
```

https://github.com/wxt-dev/wxt/blob/4f772fc0f7bdecc597a5470639cb78bd93682cac/packages/module-react/entrypoints/content/index.tsx#L25-L34

React module is following `return appInstance;` in `onMount` and `unmount` in `onRemove`. So I think it must changed like above.